qiskit/aqua/operators/legacy/base_operator.py

```diff
@@ -17,7 +17,7 @@
 from abc import ABC, abstractmethod
 
 
-class BaseOperator(ABC):
+class LegacyBaseOperator(ABC):
```

Comment:
I think `BaseOperator` should be kept as deprecated class, there's probably a lot of chemistry code of researchers fiddling with operators and it would be frustrating if all of that broke. Also are you intending to keep `LegacyBaseOperator` or is it going to be deprecated? If it is going to be deprecated we could also just keep a deprecated `BaseOperator`.

Reply:
The code cannot be deprecated as of now as Z2Symmetry capability is not present in new operator logic. When this is available then the chemsitry stack can be switched over, but until such time as it is the former operators will have to sit alongside the new. When that happens the legacy section would be deprecated and removed later as is the normal practice